web/controllers/registrationController/index.ts

```diff
@@ -52,6 +52,7 @@ export async function postTutorHandler(req: Request, res: Response) {
         if (typeof req.body.firstname == 'string' &&
             typeof req.body.lastname == 'string' &&
             typeof req.body.email == 'string' &&
+            typeof req.body.phone == 'string' || typeof req.body.phone == 'undefined' &&
```

Comment:
this won't work due to operator predescendence. It reads as (firstname && lastname && email && phone) || (phone && ..). You might want to put this or in braces.

Reply:
Thanks for alerting. Fixed now.